language/testing-infra/e2e-tests/goldens/language_e2e_testsuite::tests::data_store::borrow_after_move.exp

```diff
@@ -1,4 +1,4 @@
-Ok([TransactionOutput { write_set: WriteSet(WriteSetMut { write_set: [(AccessPath { address: 5455a8193f1dfc1b113ecc54d067afe1, path: 01000000000000000000000000000000010b4469656d4163636f756e740b4469656d4163636f756e7400 }, Value(20f5b9d6f01a99e74c790e2f330c092fa05455a8193f1dfc1b113ecc54d067afe1015455a8193f1dfc1b113ecc54d067afe1015455a8193f1dfc1b113ecc54d067afe100000000000000001800000000000000005455a8193f1dfc1b113ecc54d067afe100000000000000001800000000000000005455a8193f1dfc1b113ecc54d067afe10b00000000000000)), (AccessPath { address: 5455a8193f1dfc1b113ecc54d067afe1, path: 005455a8193f1dfc1b113ecc54d067afe1014d }, Value(a11ceb0b01000000090100040204040308190521140735420877200a9701050c9c014b0de7010200000101000201000003000100000402010000050001000006000100010800040001060c0002060c03010608000105010708000103014d065369676e657202543109626f72726f775f7431096368616e67655f74310972656d6f76655f74310a7075626c6973685f743101760a616464726573735f6f665455a8193f1dfc1b113ecc54d067afe10000000000000000000000000000000100020107030001010003050b0011042b000c01020101010005090b0011042a000c020b010b020f0015020201010006060b0011042c0013000c010203010001050b0006030000000000000012002d0002000000))] }), events: [], gas_used: 8, status: Keep(EXECUTED) }])
+Ok([TransactionOutput { write_set: WriteSet(WriteSetMut { write_set: [(AccessPath { address: 5455a8193f1dfc1b113ecc54d067afe1, path: 01000000000000000000000000000000010b4469656d4163636f756e740b4469656d4163636f756e7400 }, Value(20f5b9d6f01a99e74c790e2f330c092fa05455a8193f1dfc1b113ecc54d067afe1015455a8193f1dfc1b113ecc54d067afe1015455a8193f1dfc1b113ecc54d067afe100000000000000001800000000000000005455a8193f1dfc1b113ecc54d067afe100000000000000001800000000000000005455a8193f1dfc1b113ecc54d067afe10b00000000000000)), (AccessPath { address: 5455a8193f1dfc1b113ecc54d067afe1, path: 005455a8193f1dfc1b113ecc54d067afe1014d }, Value(a11ceb0b02000000090100040204040308190521140735420877200a9701050c9c014b0de7010200000101000201000003000100000402010000050001000006000100010800040001060c0002060c03010608000105010708000103014d065369676e657202543109626f72726f775f7431096368616e67655f74310972656d6f76655f74310a7075626c6973685f743101760a616464726573735f6f665455a8193f1dfc1b113ecc54d067afe10000000000000000000000000000000100020107030001010003050b0011042b000c01020101010005090b0011042a000c020b010b020f0015020201010006060b0011042c0013000c010203010001050b0006030000000000000012002d0002000000))] }), events: [], gas_used: 8, status: Keep(EXECUTED) }])
```

Comment:
E2E tests golden files updated because the version number in the `CompiledModule` is bumped from 1 to 2.

Reply:
With 13661c2, not only the version number is bumped to 2, the `flags` byte is changed as well.